src/main/java/com/redhat/labs/lodestar/exception/mapper/GitLabServiceResponseMapper.java

```diff
@@ -9,23 +9,30 @@
 import org.eclipse.microprofile.rest.client.ext.ResponseExceptionMapper;
 
 @Priority(4000)
-public class GitLabServiceResponseMapper implements 
-             ResponseExceptionMapper<RuntimeException> {
-  @Override
-  public RuntimeException toThrowable(Response response) {
-    int status = response.getStatus();
+public class GitLabServiceResponseMapper implements ResponseExceptionMapper<RuntimeException> {
+    @Override
+    public RuntimeException toThrowable(Response response) {
+        int status = response.getStatus();
```

Comment:
should we log out the error + stack here. It seems like we get no error messaging in the git-api

Reply:
What I found when debugging this, is that `src/main/java/com/redhat/labs/lodestar/exception/mapper/ApiExceptionMapper.java` was handling the exception instead of this mapper.  Once I removed the other one, the messages from GitLab were coming back as expected.  I don't have an issue also logging the stack trace as well.

Reply:
Actually, just retested this and it does log the stack trace with the message from the GitLab API.